S/spdlog/build_tarballs.jl

```diff
@@ -0,0 +1,31 @@
+using BinaryBuilder
+
+name = "spdlog"
+version = v"1.14.1"
+
+sources = [GitSource("https://github.com/gabime/spdlog.git", "27cb4c76708608465c413f6d0e6b8d99a4d84302")]
+
+script = raw"""
+cd ${WORKSPACE}/srcdir/spdlog*
+mkdir build
+cd build
+cmake -S .. -B . \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_PREFIX_PATH=${prefix} \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DSPDLOG_FMT_EXTERNAL=ON \
+    -DSPDLOG_BUILD_SHARED=ON \
+    -DSPDLOG_BUILD_PIC=ON \
+    -DSPDLOG_BUILD_EXAMPLE=OFF
+make -j${nproc} install
+"""
+
+platforms = expand_cxxstring_abis(supported_platforms())
+
+products = [LibraryProduct("libspdlog", :libspdlog)]
+
+dependencies = [Dependency("Fmt_jll")]
```

Comment:
Based on  #9110 you have to specify the compat bound on this package (and using compat bounds is a good idea regardless)

Reply:
I didn't add one because I couldn't see anywhere in the spdlog documentation that said it only had compatibility with some version(s) of libfmt.

Reply:
Okay. How do I determine what compat bounds would be appropriate then?

Reply:
According to #9110 v9 isn't compatible with v11.  So pick whatever you want.

Reply:
```suggestion
dependencies = [Dependency("Fmt_jll"; compat="11")]
```